app-shells/hstr/hstr-1.24.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="Shell history suggest box"
+DESCRIPTION="Easily view, navigate, search and use your command history with \
+shell history suggest box for Bash and ZSH."
+HOMEPAGE="http://me.mindforger.com/projects/hh.html"
+COPYRIGHT="2014 Martin Dvorak"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/dvorka/hstr/releases/download/$portVersion/hh-$portVersion-src.tgz"
+CHECKSUM_SHA256="1dc07cf0bdbb5ff36ee85a45f929b56a1f9776e63d51f84fc8067377ced11171"
+SOURCE_DIR="hstr"
+PATCHES="hstr-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+fi
+
+PROVIDES="
+	hstr$secondaryArchSuffix = $portVersion
+	cmd:hh$commandSuffix = $portVersion
+	cmd:hstr$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libncursesw$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage hstr$secondaryArchSuffix \
+	"$binDir"/hh$secondaryArchSuffix
```

Comment:
```diff
-	"$binDir"/hh$secondaryArchSuffix
+	"$commandBinDir"/hh
```
You should also define the [`commandBinDir`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable) variable and use it where appropriate, in particular in the call to `runConfigure`.